packages/core/src/channels/agent-channels.ts

```
@@ -21,6 +21,8 @@ import { createTool } from '../tools/tool';
 
 import { chatModule, getChatModule } from './chat-lazy';
 import { resolveSlackTopLevelThreadId } from './compat/slack';
+import { resolveSlackMentions } from './slack-mentions';
```

ESLint

[error] `./slack-mentions` import should occur after import of `./processor`

(import/order)